### DISCUSSION TITLE
go.d/ping : how do I force IPv4 or IPv6 on a domain name in Host?

### DISCUSSION THREAD

**Discussion by cayenne17** [netdata/netdata#16060] [Q&A] [answered]
Hello,
I haven't found in the [documentation](https://learn.netdata.cloud/docs/data-collection/synthetic-checks/network-latency-check) how to force IPv4 or IPv6 on a domain name ?

**Comment by ilyam8** [comment]
Hey, @cayenne17. There is no such configuration option, but we can add it because the [package we use has this functionality](https://github.com/prometheus-community/pro-bing/blob/a02e905a7d308d14663d6232e356f6003c73de8f/ping.go#L370-L383).

**Reply by ilyam8** [reply, marked as answer]
Added in https://github.com/netdata/go.d.plugin/pull/1348

**Reply by cayenne17** [reply]
Thank you @ilyam8 what speed !

It's coming in the next version of Netdata, isn't it?

**Reply by ilyam8** [reply]
Next nightly release (tomorrow).
